### DISCUSSION TITLE
Otra version de la pag con otros colores.

### DISCUSSION THREAD

**Discussion by Ivanovicc** [guillermo-k/App_restaurant#70] [Ideas]
Probé otro sistema ademas de las tablas, ustedes díganme que les parece, pase a oscura la pag por el tema de la vista y la luz azul de las pantallas. También solucione el problema con los link del nav, me había olvidado de ponerles el href solamente. Funcionan correctamente.

![captura](https://user-images.githubusercontent.com/64761565/168412291-d1cf8e7d-6e40-4b9e-badd-a0f7cf741c92.png)
![captura2](https://user-images.githubusercontent.com/64761565/168412293-994ef056-2561-43a2-90ed-db43913c439b.png)

**Comment by guillermo-k** [comment]
Me gustó.
Pregunto, que pasa en el caso de haber, no se, 40 mesas? No se va a extender demasiado para abajo?
Cómo vez implementar que cambien las dimensiones de las "cajas" que contienen cada mesa, dependiendo la cantidad de estás? Cómo para que si son muchas mesas, entren más en pantalla.

O tal vez, incluir una lista de las mesas con link a un ancla dentro de la misma página en la mesa correspondiente.




Tema aparte, me parece que si la mesa no cuenta con nigún pedido, no debería mostrarse el botón de "ver". Que piensan ustedes?

**Comment by Ivanovicc** [comment]
Lo de las dimensiones no quedaría así, la idea seria que entren en una fila 5 o 6, como para que sea legible también, lo de la listas también lo sigo pensando y lo he estado intentando con un modal para ver cada pedido de cada mesa, la pagina de editar no cargaba el pedido verdad? como para cargar un pedido y mas o menos tener una idea de como se tendría que ver cada carta con sus pedido. 

Me gusta la idea de que si no tiene ningún pedido no pueda ver el botón, o que diga desocupado o disponible mejor.

Le mando un push?, estuve viendo y no rompí nada mas creo jaja. Lo revise como 5 veces anoche y hoy jajaja.

**Comment by gianfranco-s** [comment]
Uhhh no había visto esto! Excelente!
